### ISSUE TITLE
Disable GPU (--disable-gpu) option missing in command line help

### ISSUE THREAD

**Issue by jrast** [Microsoft/vscode#13706]
The command line help (code --help) does not show the option for disabling GPU rendering (--disable-gpu)

- VSCode Version: 1.6.0
- OS Version: Debian Jessie

Steps to Reproduce:

1. Open Terminal
2. Type "code --help"

Output on my system:

```
Visual Studio Code 1.6.0
Usage: code [options] [paths...]

Options:
  -d, --diff                  Open a diff editor. Requires to pass two file paths as arguments.
  -g, --goto                  Open the file at path at the line and column (add :line[:column] to path).
  --locale <locale>           The locale to use (e.g. en-US or zh-TW).
  -n, --new-window            Force a new instance of Code.
  -p, --performance           Start with the 'Developer: Startup Performance' command enabled.
  -r, --reuse-window          Force opening a file or folder in the last active window.
  --user-data-dir <dir>       Specifies the directory that user data is kept in, useful when running as root.
  --verbose                   Print verbose output (implies --wait).
  -w, --wait                  Wait for the window to be closed before returning.
  --extensionHomePath         Set the root path for extensions.
  --list-extensions           List the installed extensions.
  --install-extension <ext>   Installs an extension.
  --uninstall-extension <ext> Uninstalls an extension.
  --disable-extensions        Disable all installed extensions.
  -v, --version               Print version.
  -h, --help                  Print usage.
```

**Comment by joaomoreno** [comment]
Care to submit a PR?

**Comment by joaomoreno** [comment]
Fixed by #13760
